### PR TITLE
conversation #2

### PR DESCRIPTION
**Pull request by tqhung04**

### CODE REVIEW

**Comment by tqhung04** [comment]
ready

File: app/controllers/conversations_controller.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 class ConversationsController < ApplicationController
-  before_action :logged_in_user, :get_current_conversation, :allowed?,
-    :get_conversations, :get_last_messages, only: %i(show)
+  before_action :get_conversations, only: %i(index show)
+  before_action :logged_in_user, :get_current_conversation,
+    :allowed?, only: %i(show)
 
-  def index; end
+  def index
+    redirect_to action: "show", id: @conversations.last.id
```
Comment (baclv) [inline app/controllers/conversations_controller.rb]: dùng symbol bạn nhé

**Comment by tqhung04** [comment]
ready

**Comment by baclv** [comment]
merged

**Pull request merged**